.github/workflows/label-analysis.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: "Label analysis"
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled]
+    branches:
+      - main
+      - 7.[0-9][0-9].x
```

Comment:
Because it's the name of our release branches (`7.50.x` for instance if I'm correct)
You are right I should probably put 6 as well (only main and official release branches were my objective)

Reply:
_Ugh_ duh. I think while we are making the changes we should go ahead and include `6` and given that we are half-way to triple digits in the minor version we should make sure that's also supported?